README.md

```diff
@@ -196,59 +221,31 @@ If you want to use a custom database, you will need to create a Sourmash sketch
 If you have a single FASTA file with _one genome_ per record:
 
 ```bash
-sourmash sketch dna -f -p k=31,scaled=1000,abund --singleton <your multi-FASTA file> -o training_database.sig.zip
+yacht sketch ref --infile <path to your multi-FASTA file> --kmer 31 --scaled 1000 --outfile training_database.sig.zip
 ```
 
 If you have a directory of FASTA files, one per genome:
 
 ```bash
-## Method 1 (suggested)
-# put all full paths of FASTA/FASTQ file into a file, one path per line
-find <path of foler containg FASTA/FASTQ files> > dataset.csv
-sourmash sketch fromfile dataset.csv -p dna,k=31,scaled=1000,abund -o ../training_database.sig.zip
-# cd back to YACHT
-
-## Method 2
-# cd into the relevant directory
-sourmash sketch dna -f -p k=31,scaled=1000,abund *.fasta -o ../training_database.sig.zip
-# cd back to YACHT
+yacht sketch ref --infile <path of foler containg FASTA/FASTQ files> --kmer 31 --scaled 1000 --outfile training_database.sig.zip
 ```
 
 </br>
 
 
-### Creating sketches of your sample
+### Creating sketches of your sample (yacht sketch sample)
 
 Creating a sketch of your sample metagenome is an essential step in the YACHT workflow. This process involves using the same k-mer size and scale factor that were used for the reference database. You can use the following commands to implement this step:
 
 ```bash
 # For a single-end FASTA/Q file
-sourmash sketch dna -f -p k=31,scaled=1000,abund -o sample.sig.zip <input FASTA/Q file>
+yacht sample --infile <input FASTA/Q file> --kmer 31 --scaled 1000 --outfile sample.sig.zip
```

Comment:
Same comment before re: not deleting the sourmash command. Eg. you could just after this bit of code say something like "This is equivalent to `<insert sourmash command>`."

Reply:
This has been fixed via the above response.